static/i18n/en/app.json

```diff
@@ -179,11 +179,20 @@
   },
   "emptyState": {
     "dashboard": {
-      "title": "Install apps or add accounts",
-      "desc": "Open the Manager to install apps or add accounts if your Ledger device already has apps installed",
+      "title": "Install an app on my device",
+      "desc": "You need to install apps on your device to add accounts in Ledger Live. Go to the manager and install apps on your device.",
       "buttons": {
-        "addAccount": "Add accounts",
-        "installApp": "Open Manager"
+        "installApp": "Install Apps",
+        "help": "Help"
+      }
+    },
+    "accounts":{
+      "title": "You don’t have any accounts…",
+      "desc": "Click on “Add account” to stat adding accounts in Ledger Live",
```

Comment:
Add an account to start managing your crypto assets. You should have installed the apps for your crypto assets on your device.